### DISCUSSION TITLE
Is it possible to disable/remove subscribers messages from /metions tab?

### DISCUSSION THREAD

**Discussion by 001KeV** [Chatterino/chatterino2#3992] [Q&A] [answered]
Hello, 
I would like to know if it is possible to disable/remove subscribers's highlighted messages from the /mentions tab.
Just because I would like to see only the users that I added in, in the "Highlights -> Users" settings. So, no subscribers's messages basically. I couldn't notice any setting for that.

Thank you in advance.

**Comment by Mm2PL** [comment, marked as answer]
You could try applying a filter to your `/mentions` tab. You can create one in the settings. [The wiki](https://wiki.chatterino.com/Filters/) has an introduction to them.

**Reply by Felanbird** [reply]
The filter in question you want is just `!flags.sub_message`

**Reply by 001KeV** [reply]
It worked perfectly. Thank you both.
